torch/_inductor/config.py

```diff
@@ -582,6 +582,9 @@ def use_autoheuristic(name: str) -> bool:
 # how many nodes to allow into a single fusion
 max_fusion_size = 64
 
+# how far a node may be fused with another node
+max_fusion_proximity = 64
```

Comment:
this name is not accurate - can we rename it ? `max_fusion_buffer_group_pairwise_attempts` or something.